scaffolder-templates/basic-workflow/template.yaml

```diff
@@ -299,6 +299,7 @@ spec:
         files:
           - gitops/kustomize/base/sonataflow-patch-persistence.yaml
     - id: publishGitOps
+      if: ${{ parameters.CI == 'tekton_argocd' }}
```

Comment:
as you prefer, but in this case the developer will not have any pre-defined way to deploy its application, even with manual execution of CLI commands